test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+    
+    Name name;
+    
+    @Before
+    public void setUp() throws Exception {
+        name = new Name("name n");
+    }
+
+    @Test
+    public final void test_isSimilar_sameString() throws IllegalValueException {
```

Comment:
you can eliminate the word `test` from the method name; 
the test name should have 3 parts `<method under test>_<which scenario/test case>_<expected return value>`